src/site/stages/build/plugins/modify-dom/check-broken-links/tests/index.unit.spec.js

```diff
@@ -0,0 +1,26 @@
+const brokenLinkChecker = require('../index');
+const { expect } = require('chai');
+
+describe('broken link checker', () => {
+  it('determines if homepage is broken', () => {
+    let brokenPages = [{ path: '/', linkErrors: new Array(1) }];
+
+    expect(brokenLinkChecker.deriveIsHomepageBroken(brokenPages)).to.be.true;
+
+    brokenPages = [{ path: '/not-homepage', linkErrors: new Array(1) }];
+
+    expect(brokenLinkChecker.deriveIsHomepageBroken(brokenPages)).to.be.false;
+  });
```

Comment:
I would recommend splitting this into two tests (but I'm not going to block the PR for this), one for `deriveIsHomepageBroken returns true when path for broken link is '/'` and then a second for `deriveIsHomepageBroken returns false when path for broken link is not '/'`. Just for clarity and keeping our tests as simple as possible.

Reply:
Good idea, I was being lazy! Just pushed a commit to do that

Reply:
Thank you!